src/app.rs

```diff
@@ -215,8 +215,7 @@ pub fn build() -> App<'static, 'static> {
                 .short("i")
                 .long("inode")
                 .multiple(true)
-                .conflicts_with("blocks")
-                .help("Display inode and file names"),
+                .help("Display inode(Linux only)"),
```

Comment:
Use something like:

`print the index number of each file`

Reply:
I updated the help msg, but using the `Display`,  for the consistent of other help msgs.